hamilton/execution/graph_functions.py

```diff
@@ -250,7 +258,7 @@ def execute_lifecycle_for_node(
                     else:
                         result = node_(**kwargs)
                     
-                    return error, result, success, pre_node_execute_errored
+                    return result#, success, pre_node_execute_errored
```

Comment:
I am struggling to capture these, since now ```ray.remote``` is returning the reference that needs to be passed into ```ray.get``` to get the results of the 'sandwich' function that helps us get the hooks through. I feel like there a step missing in ```RayGraphAdapter.build_result()``` where these would be unpacked and passed back to ```execute_subdag```

Reply:
so the way it works right now is that we get ray object references back. The `.remote(**kwargs)` call on the ray.remote function is where Ray figures out what to resolve. 

I think we want the `num_returns` construct here in that case.

```
@ray.remote(num_gpus=1, max_calls=1, num_returns=2)
def f():
    return 1, 2

f_with_2_gpus = f.options(num_gpus=2)
object_refs = f_with_2_gpus.remote()
assert ray.get(object_refs) == [1, 2]
```
i.e. you'll then get separate object references for each return value that you can wire through.